src/TagManager.lua

```diff
@@ -224,26 +252,28 @@ function TagManager:_doUpdateStore()
 	local tags: { Tag } = {}
 	local groups: { [string]: boolean } = {}
 	local sel = Selection:Get()
-	local tagNamesSeen: { [string]: boolean } = {}
+	local tagNamesSeen: { [string]: string } = {}
 
-	if self.tagsFolder then
-		for _, inst in pairs(self.tagsFolder:GetChildren()) do
+	local function update(folder)
+		for _, inst in pairs(folder:GetChildren()) do
 			if not inst:IsA("Configuration") then
 				continue
 			end
 			if tagNamesSeen[inst.Name] then
 				if not self._gaveDuplicateWarningsFor[inst.Name] then
 					warn(
 						string.format(
-							"Multiple tags in ServerStorage.TagList are named %q, consider removing the duplicates.",
+							"Multiple tags in %s and %s are named %q, consider removing the duplicates.",
```

Comment:
Since there's some deliberate override logic here, I'm thinking this should only warn if the duplicate tags are both in the same folder.